static/js/add_subscribers_pill.js

```diff
@@ -59,6 +59,21 @@ export function create({$pill_container, get_potential_subscribers}) {
 
     set_up_pill_typeahead({pill_widget, $pill_container, get_users});
 
+    const $pill_widget_input = $pill_container.find(".input");
+    const $pill_widget_button = $pill_container.parent().find(".add-subscriber-button");
+    // Disable the add button first time the pill container is created.
+    $pill_widget_button.prop("disabled", true);
+
+    // If all the pills are removed, disable the add button.
+    pill_widget.onPillRemove(() =>
+        $pill_widget_button.prop("disabled", pill_widget.items().length === 0),
+    );
+    // Disable the add button when there is no pending text that can be converted
+    // into a pill.
+    $pill_widget_input.on("input", () =>
+        $pill_widget_button.prop("disabled", !pill_widget.is_pending()),
```

Comment:
I noticed one small bug which i think is due to this code. The button is disabled even if I have a pill inside.
![add-sub-disable-bug](https://user-images.githubusercontent.com/35494118/216604288-50d91db1-a754-4ab2-9309-11c1c4d4e8db.gif)

Reply:
Fixed the bug. Thanks. Ready for review.